server/tests/common/client.py

```diff
@@ -130,7 +130,7 @@ async def _do_request(self, req: bytes) -> bytes:
         return rep.content
 
     @asynccontextmanager
-    async def events_listen(
+    async def events_listen(  # pyright: ignore [reportIncompatibleMethodOverride]
```

Comment:
Why incompatible, can you add a comment above ?

Reply:
The return type of `events_listen` here is not compatible with that of the base class `events_listen`:
```
client.py:133:15 - error: Method "events_listen" overrides class "BaseAuthenticatedRpcClient" in an incompatible manner
    Return type mismatch: base method returns type "Coroutine[Any, Any, Rep]", override returns type "_AsyncGeneratorContextManager[EventsListenSSE]"
      "_AsyncGeneratorContextManager[EventsListenSSE]" is incompatible with "Coroutine[Any, Any, Rep]" (reportIncompatibleMethodOverride)
```